### PR TITLE
Loops: while and for

### PR DESCRIPTION
**Pull request by alvar-o**
This is my first pull request. Comments appreciated!

### CODE REVIEW

**Comment by alvar-o** [comment]
> [CLA assistant check](https://cla-assistant.io/javascript-tutorial/pt.javascript.info?pullRequest=50) 
> Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/javascript-tutorial/pt.javascript.info?pullRequest=50) before we can accept your contribution.
> 
> **Álvaro Antonio** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/javascript-tutorial/pt.javascript.info?pullRequest=50) it.

Hey, I followed instructions to set my email address in Git and signed the CLA. Do I have to submit a new PR to make it work?

File: 1-js/02-first-steps/12-while-for/7-list-primes/solution.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
-There are many algorithms for this task.
+Há muitos algotimos para esta tarefa.
 
-Let's use a nested loop:
+Vamos utilizar um loop dentro de outro (isto é, aninhado):
```
Comment (Mr-Saxobeat) [inline 1-js/02-first-steps/12-while-for/7-list-primes/solution.md]: I think we need to add the "nested loop" term here, inside parenthesis.

File: 1-js/02-first-steps/12-while-for/7-list-primes/solution.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
 ```js
-For each i in the interval {
-  check if i has a divisor from 1..i
-  if yes => the value is not a prime
-  if no => the value is a prime, show it
+Para cada i no intervalo {
+  testar se  i tem um divisor de 1 a i
```
Comment (Mr-Saxobeat) [inline 1-js/02-first-steps/12-while-for/7-list-primes/solution.md]: * "testar" in simple present form.

**Comment by jonnathan-ls** [comment]
@alvar-o 

Are you still available to proceed with the translation?

**Pull request closed without merging**